package.json

```diff
@@ -306,6 +306,7 @@
     "octokit": "^2.1.0",
     "octokit-plugin-create-pull-request": "^3.11.0",
     "prop-types": "^15.6.1",
+    "querystring-es3": "^0.2.1",
```

Comment:
needed because we're still relying on uswds v1 grid styles while we get off of Formation the repo.